front/document_item.form.php

```diff
@@ -44,6 +44,11 @@
 
 if (isset($_POST["add"])) {
    $document_item->check(-1, CREATE, $_POST);
+
+   // #1476 - Inject ID of the actual user to known who attach an already existing document
+   // to another item
+   $_POST['users_id'] = $_SESSION["glpiID"];
```

Comment:
last review, other part is good for me.
We prefer to use Class::PrepareInputForAdd instead populating data in front/ files.
So, API queries will have also the good users_id filled